test/template-tests/Test-AzureRMTemplate.ps1

```diff
@@ -182,7 +183,7 @@ Each test script has access to a set of well-known variables:
         #*Test-Group (executes a group of tests)
         function Test-Group {                
             $testQueue = [Collections.Queue]::new(@($GroupName))
-            while ($testQueue.Count) {
+            :nextTest while ($testQueue.Count) {
```

Comment:
nit - does the label need to be on the same line as the while stmt?

Reply:
_kinda_ technically it just needs to preceed a keyword.  The point of using it here is to avoid extra processing of the current test if we're skipping it.